### DISCUSSION TITLE
{Feature Request] a way to properly enter a transaction that has been made with monthlyy payments

### DISCUSSION THREAD

**Discussion by FSNaval** [firefly-iii/firefly-iii#7914] [General]
When a purchase is made that concerns paying a big amount of money, for example you buy a tv with your credit card for 3000 EUROS, many banks give you the opportunity to pay this amount gradually with monthly installments.

It would be desirable if such a feature be implemented in firefly, that is, when you make a purchase that you can pay in many smaller installments, an option could be presented that you could declare in firefly how many installments the purchase has and firefly automatically to split the given amount to these installments.

see photo in this link: https://www.dropbox.com/scl/fi/dkunqxi74lg9fy65jciq0/IMG_1036.png?rlkey=zvh7549b16kq7fjiapde4yl5f&dl=0

**Comment by JC5** [comment]
You can make a recurring transaction with a limited number of runs.

**Reply by FSNaval** [reply]
Yes, that is a way.

But when i create a report from firefly to make a prediction for the transactions of the next six months for example, these recurring transactions are not generated and are not taken into account from the software and are not shown in the report.

That is, i see a flat line extending all the way from today's date for the next six months, when in fact i should be seeing lines going up and down (income transactions and expenses transactions taken into account).

**Reply by diamant-x** [reply]
I don't think there's much functionality currently in FF3 for forecasts other than Bills.
Your use case is basically creating a liability for which you need to pay, so at least you can reflect that 'money you will end up paying' in your net worth calculation in FF3.
